### PR TITLE
Fix (multiedit closing fixed)

### PR DESCRIPTION
**Pull request by Aca-jov**
SD-3246, SD-3270

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/workqueue/workqueue.js [hunk 1]
```diff
@@ -78,7 +78,15 @@ function WorkqueueCtrl($scope, $route, workqueue, multiEdit, superdesk, lock) {
...
     $scope.closeItem = function(item) {
-        lock.unlock(item).then(updateWorkqueue);
+        multiEdit.items = _.without(multiEdit.items, _.find(multiEdit.items, {article: item._id}));
+        lock.unlock(item);
+        if (authoringWorkspace.item && item._id === authoringWorkspace.item._id){
+            authoringWorkspace.close();
+        }
+        if ($scope.isMultiedit && multiEdit.items.length == 0){
+            $scope.isMultiedit = false;
+            $location.url('/workspace/monitoring');
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/workqueue/workqueue.js]: Instead of hardcoding it multiple can we extract it to a method?

**Comment by sivakuna-aap** [comment]
:+1: except the minor comment

**Comment by plamut** [comment]
Please add/update tests and docstrings for the modified methods, this represents an essential part of a minimal viable pull request as was defined in one of the more recent strategic meetings.

Please also pay more attention to commit messages, this has been mentioned several times by now. Too broad or vague messages really make it harder to review commit history later. Every commit should also include a reference to the ticket number, this helps with Jira integration (displaying branch/build statuses directly aside the ticket description).

**Comment by pavlovicnemanja** [comment]
:+1:

**Pull request merged**